macros/WebExtExamples.ejs

```diff
@@ -1,16 +1,47 @@
 <%
 
-var allExamples = mdn.fetchJSONResource("https://raw.githubusercontent.com/mdn/webextensions-examples/master/examples.json");
+/*
+The "examples.json" file from the https://github.com/mdn/webextensions-examples
+repository contains information about the examples living in that repository.
 
-var examplesInAPIHeading = $0 || "h3";
+This macro must be embedded in a WebExtensions API page, either of:
 
+* a "module" page (for example: "/en-US/Add-ons/WebExtensions/API/alarms" or "/en-US/Add-ons/WebExtensions/API/tabs")
+* an "API" page (for example: "/en-US/Add-ons/WebExtensions/API/alarms/create" or "/en-US/Add-ons/WebExtensions/API/tabs/executeScript")
+
+If it's embedded in a module page, it fetches all the examples that use any
+of the APIs in the given module, and outputs a list of links to those examples.
+
+If it's embedded in an API page, it fetches all the examples that use the
```

Comment:
This suggests to me that this macro could be further split into a version for module pages and a version for API pages, but I'm guessing that would require a lot more page editing, and wouldn't be worth it at this time.